### PR TITLE
MULE-10975: AbstractAggregator eventGroups object store ignores the

### PR DESCRIPTION
**Pull request by elrodro83**
object store configuration in the registry

### CODE REVIEW

File: core/src/main/java/org/mule/routing/AbstractAggregator.java [hunk 1]
```diff
@@ -148,27 +148,20 @@ protected Factory internalEventsGroupsObjectStoreFactory()
             @Override
             public Object create()
             {
-                try
+                ObjectStore objectStore;
+                if (persistentStores)
+                {
+                    objectStore = muleContext.getRegistry().lookupObject(OBJECT_STORE_DEFAULT_PERSISTENT_NAME);
```
Comment (pablolagreca) [inline core/src/main/java/org/mule/routing/AbstractAggregator.java]: This must be done using the ObjectStoreManager, just ask for a persistent object store.

**Comment by pablolagreca** [comment]
Test case?

**Comment by elrodro83** [comment]
The test case is in the cluster module, the pr is in ee

File: core/src/main/java/org/mule/routing/AbstractAggregator.java (hunk 1 above)
Comment (elrodro83) [inline core/src/main/java/org/mule/routing/AbstractAggregator.java]: The thing is that in order to support the configurable objectStore, the partitions have to be handled a bit differently. The obtained object will be wrapped in a ProvidedPartitionableObjectStoreWrapper to handle that logic (the creation of the corresponding partitions). The type returned when using the ObjectStoreManager does not support this.

**Pull request merged**